android/src/main/java/com/swmansion/gesturehandler/core/RotationGestureHandler.kt

```diff
@@ -59,7 +59,8 @@ class RotationGestureHandler : GestureHandler<RotationGestureHandler>() {
       anchorY = point.y
     }
     if (sourceEvent.actionMasked == MotionEvent.ACTION_UP) {
-      if (state == STATE_ACTIVE) {
+      var activePointers = sourceEvent.pointerCount
+      if (state == STATE_ACTIVE && activePointers == 0) {
```

Comment:
Wouldn't this be enough?

```suggestion
      if (state == STATE_ACTIVE && sourceEvent.pointerCount == 0) {
```